src/Symfony/Component/Mailer/Bridge/Sendinblue/CHANGELOG.md

```diff
@@ -0,0 +1,7 @@
+CHANGELOG
+=========
+
+5.3.0
```

Comment:
Could be 5.2 as I think we can add it in 5.2 in time.

Reply:
I have changed this also for symfony-docs and recipe.